lib/Graph/Graph.cpp

```diff
@@ -2993,3 +2993,15 @@ bool Function::verify() const {
   }
   return isValid;
 }
+
+/// If \p PH is an output placeholder, \returns the SaveNode.
+SaveNode *glow::getOutputSave(Placeholder *PH) {
+  for (auto &use : PH->getUsers()) {
+    if (auto *save = llvm::dyn_cast<SaveNode>(use.getUser())) {
+      if (save->getPlaceholder() == PH) {
```

Comment:
can a placeholder be used by multiple Save nodes (after we clone function for example)?

Reply:
Yeah, this function likely needs to take the Function under from you want to find the `SaveNode`.